packages/syntax-core/src/WordConfetti/WordConfetti.tsx

```diff
@@ -108,9 +120,8 @@ const WordConfetti = forwardRef<HTMLDivElement, WordConfettiProps>(
       () =>
         words.map((word) => ({
           text: word,
-          backgroundColor:
-            themeBackgroundColors[theme][Math.floor(Math.random() * 3)],
-          rotation: degreeOfTiltOptions[Math.floor(Math.random() * 9)],
+          backgroundColor: themeBackgroundColors[theme][hashCode(word) % 3],
```

Comment:
@jliotta thinking about these numbers `3` and `5`, we should probably base those on the length of options instead. See https://github.com/Cambly/syntax/pull/559 for a solution